src/sonic-yang-models/yang-models/sonic-mclag.yang

```diff
@@ -0,0 +1,167 @@
+module sonic-mclag {
+    namespace "http://github.com/Azure/sonic-mclag";
+    prefix smclag;
+    yang-version 1.1;
+
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-portchannel {
+		prefix lag;
+    }
+
+    import sonic-vlan {
+        prefix vlan;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC MCLAG";
+
+    revision 2019-10-01 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-mclag {
+
+        container MCLAG_DOMAIN {
+
+            list MCLAG_DOMAIN_LIST { 
+                key "domain_id";
+                max-elements 1;
+
+                leaf domain_id {
+                    description  "MCLAG Domain ID";
```

Comment:
>   [](http://example.com/codeflow?start=31&length=2)

Use one blank instead of two, if you don't need align it with other column.

Reply:
addressed this comment